README.md

```diff
@@ -1,32 +1,33 @@
-![](assets/images/logo.png)
+![](static/images/logo.png)
 
-> A documentation website for Momentum Mod, running on [Jekyll](https://jekyllrb.com/) and using the [Minimal Mistakes](https://github.com/mmistakes/minimal-mistakes) theme.
+> A documentation website for Momentum Mod, running on [Hugo](https://gohugo.io/) and using the [Hugo Book](https://github.com/alex-shpak/hugo-book) theme.
 
 All documentation is kept up with the game's development branch not our current release version on Steam.
 
 ## What This Is
 
 This repo is an open source collection of documentation for various aspects of the game Momentum Mod. This includes (but is not exactly limited to):
 
- * Documenting Momentum's console commands
- * Documenting Momentum's console variables
- * Documenting Momentum's custom entities
- * Guides for using Momentum Mod or related tools (zone tools, custom entities, etc)
- * Guides for mapping (**related to Momentum and/or its gametypes**)
+- Documenting Momentum's console commands
+- Documenting Momentum's console variables
+- Documenting Momentum's custom entities
+- Guides for using Momentum Mod or related tools (zone tools, custom entities, etc)
+- Guides for mapping (**related to Momentum and/or its gametypes**)
 
 ## What This Isn't
 
 This repo and website should pertain to being **only** about the game Momentum Mod and its related gametypes (Surf, Bhop, RJ, KZ, etc).
 
 This means we do **NOT** want:
- * General guides about Hammer (this isn't [TopHattWaffle](https://www.youtube.com/user/rocketrascal34) / [Valve Dev Wiki](https://developer.valvesoftware.com/wiki/Category:Level_Design))
- * Off-topic articles (this isn't [Medium](https://medium.com/))
- * Map-specific articles (this isn't [a wikia](https://www.fandom.com/))
+
+- General guides about Hammer (this isn't [TopHattWaffle](https://www.youtube.com/user/rocketrascal34) / [Valve Dev Wiki](https://developer.valvesoftware.com/wiki/Category:Level_Design))
+- Off-topic articles (this isn't [Medium](https://medium.com/))
+- Map-specific articles (this isn't [a wikia](https://www.fandom.com/))
 
 This may change in the future, but for now, keep it about Momentum.
 
 ## Contributing
 
 Pull Requests are highly appreciated! Just branch off of master, make your edits, and open a Pull Request into master describing what you changed. Pages could be edited directly through GitHub, or you can download your fork and edit through a program like [Visual Studio Code](https://code.visualstudio.com/).
 
-If you need a guide for how to contribute, one can be found on the [how-to page](https://docs.momentum-mod.org/guide/create-docs-page/) on the documentation site. Another good resource is to simply look at existing posts, or past commits for reference. Furthermore, another good reference could be asking around in the [Momentum Mod Discord server](https://discord.gg/momentummod).
```

Comment:
Actually, on second thought, let's keep this as the full URL, so that GitHub readme links to our site.

Reply:
It was linking to the "old site" because the main docs URL wasn't updated with this deploy yet. It'll properly link to it on GitHub and on the site when it gets deployed.

Reply:
Wait, so am I changing this or not to use a relative URL or not? I just dropped that commit and force-pushed without it.